nixos/modules/services/security/physlock.nix

```diff
@@ -124,7 +140,11 @@ in
                 ++ cfg.lockOn.extraTargets;
         serviceConfig = {
           Type = "forking";
-          ExecStart = "${pkgs.physlock}/bin/physlock -d${optionalString cfg.muteKernelMessages "m"}${optionalString cfg.disableSysRq "s"}${optionalString (cfg.lockMessage != "") " -p \"${cfg.lockMessage}\""}";
+          ExecStart = pkgs.writeScript "start-physlock.sh" ''
+            #!${pkgs.bash}/bin/bash
+
```

Comment:
```suggestion
```
I would guess this line is unnecessary since you just call the physlock binary without using any shell features, but I didn't test it.